utils/client.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
```

Comment:
O kurde to to nie wiem jak napisałeś :open_mouth:

Reply:
Cały plik w sensie xd

Reply:
O kurde
rzeczywiście. Musiało mi coś w tsie popsuć się. Dzięki za czujne oko